test/test.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
The name of the file should be `test_diff.py`, to make it clear what script it is testing.